CMakeLists.txt

```
@@ -11,6 +11,7 @@ set(CMAKE_CXX_STANDARD 17)
 
 # Options
 set(INCLUDEJS_BACKEND "JavaScriptCore" CACHE STRING "The IncludeJS Engine backend")
+set(INCLUDEJS_BACKEND_JAVASCRIPTCORE_API_VERSION "" CACHE STRING "Build IncludeJS against a specific version of JavaScriptCore (empty means no preference)")
```

Is `API` here correct? Maybe something like this reads better?

```suggestion
set(INCLUDEJS_BACKEND_JAVASCRIPTCORE_VERSION "" CACHE STRING "Build IncludeJS against a specific version of JavaScriptCore (empty means no preference)")
```

---

the version is something like `2.43.4`, this is specifically the API version of webkit

---

Ah alright